src/corelibs/U2View/src/ov_assembly/AssemblyBrowser.h

```diff
@@ -247,9 +248,9 @@ private slots:
     QAction *posSelectorAction;
     PositionSelector *posSelector;
     QList<QAction *> overviewScaleTypeActions;
-    QAction *showCoordsOnRulerAction;
+    QAction *showCoordsOnRulerAction = nullptr;
```

Comment:
Why just these two initialized with `nullptr`?

Reply:
These values were not initialized at all. 
I think we should initialize all primitive type values in-place, but this again out of scope of this PR: it is already very huge.